main.go

```diff
@@ -458,7 +457,51 @@ func setupDatabase(cctx *cli.Context) (*gorm.DB, error) {
 	}
 
 	db.AutoMigrate(&Content{})
-	db.AutoMigrate(&Object{})
+
+	// Check if we need to migrate cid to hash.
+	if db.Migrator().HasColumn(&Object{}, "hash") {
+		db.AutoMigrate(&Object{})
+	} else {
+		// Migration from cid to hash is needed
```

Comment:
Manual migration only performed if we detect that the database has no hash column. Otherwise we use the auto migration.